settings.py

```diff
@@ -568,6 +568,12 @@ def get_env_map(name, default={}):
 MT_MYMEMORY_USER = None
 MT_MYMEMORY_KEY = None
 
+if 'WEBLATE_MT_MYMEMORY_ENABLED' in os.environ:
+    MACHINE_TRANSLATION_SERVICES += ('weblate.trans.machine.mymemory.MyMemoryTranslation',)
```

Comment:
I'd prefer this to be enabled only with MT_MYMEMORY_EMAIL, what is their recommendation for CAT tools, see https://mymemory.translated.net/doc/spec.php

Reply:
I thought about that as well, but according to the [docs](https://docs.weblate.org/en/weblate-2.18/admin/machine.html#mymemory) it is optional and not required.

Reply:
Indeed it's not required, but I think it's "recommended for CAT tools and high volume usage". Weblate is a CAT tool and can result in high volume usage, so I think it's reasonable to require it by default in our setup.

Alternatively we could use WEBLATE_ADMIN_EMAIL as default value.

Reply:
I've implemented the latter in 01aa629115319cd4491607f64dae4de31d66eb6b.